recipes/pisces-db/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pisces" %}
+{% set version = "0.3.2" %}
+
+
+package:
+  name: {{ name|lower }}-db
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pisces-{{ version }}.tar.gz
+  sha256: 9496b9f613348952db07121675fe32f45e23740c9d4e3885738b24ce3778ad66
+
+build:
+  skip: True    # [win]
+  number: 0
```

Comment:
This can be noarch
```suggestion
  number: 0
  noarch: python
```

Reply:
Hi, @xhochy. The upstream package specifies the platforms to be, 'Mac OS X', 'Linux/Unix' on line 39 of their [setup.py.](https://github.com/LANL-Seismoacoustics/pisces/blob/feca7afd69da84289819b09b22eb5b152818bc52/setup.py) I was unable to get the win_64 check to pass when using noarch.

Reply:
Failing windows CI with noarch packages is ok. They are only built on Linux on the feedstock. noarch is much less maintainence than non-noarch.

Reply:
@xhochy I made some changes based on what you have told me. Some checks were not successful as expected.